pricing.js

```diff
@@ -2,35 +2,43 @@ module.exports.calculateProductPrice = function (product, employee, coverageLeve
   var price = 0
   var dollarsOff = 0
 
-  switch (product.type) {
-    case 'volLife':
-      for (var i = 0; i < coverageLevels.length; i++) {
-        var coverageAmount = coverageLevels[i].coverage
+  function volLifePrice() {
```

Comment:
Good job breaking down the two product types into their own functions. Just remember, we wouldn't define these functions inside the scope of the existing function. Instead they would be defined above it as we see in the `switching-it-up` code and we would pass in whatever data they need to properly do their job.